src/topp/QualityControl.cpp

```diff
@@ -115,8 +114,11 @@ class TOPPQualityControl : public TOPPBase
     setValidFormats_("in_trafo", {"trafoXML"});
     registerTOPPSubsection_("MS2_id_rate", "MS2 ID Rate settings");
     registerFlag_("MS2_id_rate:assume_all_target", "Forces the metric to run even if target/decoy annotation is missing (accepts all pep_ids as target hits).", false);
-    registerStringOption_("out_evd", "<Path>", "", "If a Path is given, a MQEvidence txt-file will be created in this directory. If the directory does not exist, it will be created as well.", false);
-    registerStringOption_("out_msms", "<Path>", "", "If a Path is given, a MQMsms txt-file will be created in this directory. If the directory does not exist, it will be created as well.", false);
+    // MaxQuant-compatible output
+    registerTOPPSubsection_("out_txt", "Write MaxQuant-compatible .txt files");
+    registerOutputDir_("out_txt:directory", "<Path>", "", "If a Path is given, '.txt' files compatible with MaxQuant will be created in this directory. If the directory does not exist, it will be created.", false);
```

Comment:
Yeah, this is missing from the CWL output, we only get the following outputs

``` yaml
outputs:
  out:
    type: File?
    outputBinding:
      glob: $(inputs.out)
  out_cm:
    type: File?
    outputBinding:
      glob: $(inputs.out_cm)
```


While the CTD version has the following
``` xml

      <NODE name="out_txt" description="Write MaxQuant-compatible .txt files">
        <ITEM name="directory" value="" type="output-dir" description="If a Path is given, &apos;.txt&apos; files compatible with MaxQuant will be created in this directory. If the directory does not exist, it will be created." required="false" advanced="false" />
```